week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -8,9 +8,12 @@
   NOTE: don't include any element that is not a "family".
 */
 
-function colonisers() {
-
-}
+function colonisers(voyagers) {
```

Comment:
Like with the berries, this code is much more geared towards the specific test-cases we gave (which are meant to be examples) rather than the real problem to solve (described in the comments at the start). The specification of the problem talks about strings containing "family", rather than strings not being Asimov. Can you give this another go, trying to solve the problem specified in the comments, rather than the specific test-cases?